polars/polars-lazy/polars-plan/src/dsl/function_expr/strings.rs

```diff
@@ -142,34 +142,34 @@ pub(super) fn rjust(s: &Series, width: usize, fillchar: char) -> PolarsResult<Se
     Ok(ca.rjust(width, fillchar).into_series())
 }
 
-pub(super) fn strip(s: &Series, matches: Option<char>) -> PolarsResult<Series> {
+pub(super) fn strip(s: &Series, matches: &Option<String>) -> PolarsResult<Series> {
```

Comment:
It is considered good style to take the minimal indirection as argument as that make this function usable for more types. Though in this case it doesn't matter, but I'd still like to adhere to that for consistency reasons.

So this argument should be `Option<&str>`. And when calling this function we can convert like this:

```rust
let argument: Option<String>;

let argument_passed: Option<&str> = argument.as_deref();
```

Reply:
Thanks for the pointer!